itou/utils/templatetags/nav.py

```diff
@@ -152,14 +152,23 @@ def __repr__(self):
     ),
     # Employers.
     "employer-job-apps": NavItem(
-        label="Candidatures",
+        label="Candidatures reçues",
         icon="ri-draft-line",
         target=reverse("apply:list_for_siae"),
         active_view_names=["apply:list_for_siae", "apply:list_for_siae_exports"],
         matomo_event_category="offcanvasNav",
         matomo_event_name="clic",
         matomo_event_option="candidatures",
     ),
+    "employer-job-apps-sent": NavItem(
+        label="Candidatures envoyées",
+        icon="ri-folder-shared-line",
+        target=reverse("apply:list_prescriptions"),
+        active_view_names=["apply:list_prescriptions", "apply:list_prescriptions_exports"],
+        matomo_event_category="offcanvasNav",
+        matomo_event_name="clic",
+        matomo_event_option="candidatures-envoyées",
```

Comment:
```suggestion
        matomo_event_option="candidatures-envoyees",
```
Peut-être éviter les accents ?